pkg/capabilities/capabilities_test.go

```diff
@@ -31,6 +31,8 @@ func TestGet(t *testing.T) {
 		},
 	}
 
+	SetForTests(defaultCap)
```

Comment:
maybe it doesn't make any sense now the test :thinking:

Reply:
I'd tend to do something like this instead:

```diff
diff --git a/pkg/capabilities/capabilities_test.go b/pkg/capabilities/capabilities_test.go
index ea4434d0618..eba0fa73976 100644
--- a/pkg/capabilities/capabilities_test.go
+++ b/pkg/capabilities/capabilities_test.go
@@ -18,10 +18,19 @@ package capabilities
 
 import (
 	"reflect"
+	"sync"
 	"testing"
 )
 
 func TestGet(t *testing.T) {
+	// reset package vars to their defaults after we run
+	defer func() {
+		capInstance.lock.Lock()
+		defer capInstance.lock.Unlock()
+		capInstance.capabilities = nil
+		capInstance.once = sync.Once{}
+	}()
+
 	defaultCap := Capabilities{
 		AllowPrivileged: false,
 		PrivilegedSources: PrivilegedSources{
```

Reply:
yeah, that's much better
thanks